console/src/components/Services/Actions/Codegen/Codegen.js

```diff
@@ -110,24 +116,49 @@ const Codegen = ({ allActions, allTypes, currentAction }) => {
       ) {
         return null;
       }
+
       return (
-        <a
-          href={getStarterKitPath(selectedFramework)}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Button color="white" className={`${styles.add_mar_right_mid}`}>
-            Get starter kit
-          </Button>
-        </a>
+        <React.Fragment>
+          <a
+            href={getStarterKitDownloadPath(selectedFramework)}
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.add_mar_right_small}
+            title={`Download starter kit for ${selectedFramework}`}
+          >
+            <Button color="white" size="xs">
+              <DownloadIcon className={styles.add_mar_right_small} /> Starter
+              Kit
+            </Button>
+          </a>
+          <a
+            href={getStarterKitPath(selectedFramework)}
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.add_mar_right_small}
+            title={`View the starter kit for ${selectedFramework} on GitHub`}
+          >
+            <Button color="white" size="xs">
+              <GithubIcon />
+            </Button>
+          </a>
+        </React.Fragment>
+      );
+    };
+
+    const getButtons = () => {
```

Comment:
Why is the new function for it created?

https://github.com/hasura/graphql-engine-internal/issues/338#react